test/animations.html

```diff
@@ -13,6 +13,22 @@
 </head>
 
 <body>
+  <dom-module id="overlay-test-styles" theme-for="vaadin-overlay">
+    <template>
+      <style>
+        :host([animate][opening]),
+        :host([animate][closing]) {
+          animation: 50ms overlay-dummy-animation;
+        }
+
+        @keyframes overlay-dummy-animation {
+          to {
+            opacity: 1 !important; /* stylelint-disable-line keyframe-declaration-no-important */
+          }
+        }
+      </style>
+    </template>
+  </dom-module>
```

Comment:
This is an animation similar to what `vaadin-dialog` applies on the dialog overlay (where the issue was found using `vaadin-text-area` in `vaadin-dialog`).